zrp/prepare/geo_geocoder.py

```diff
@@ -202,11 +189,10 @@ def transform(self, input_data, geo, processed, replicate, save_table=True):
         no_HN_match_keys = list(set(all_keys) - set(HN_match_keys))
         
         df_no_HN = odf[odf['ZEST_KEY_LONG'].isin(no_HN_match_keys)]
-        na_match_cols = ['BLKGRPCE', 'FROMHN', 'TOHN', 'small', 'big', 'ZIP_Match_1', 'ZIP_Match_2','NEW_SUPER_ZIP', 'ZIP_Match']
-        df_no_HN[na_match_cols] = None
-        
+        na_match_cols = ['BLKGRPCE', 'FROMHN', 'TOHN', 'small', 'big', 'ZIP_Match_1', 'ZIP_Match_2','NEW_SUPER_ZIP', 'ZIP_Match']      
```

Comment:
Do we still use 'small' or 'big'? Can they or any other variables from this list be removed?

Reply:
These lines are frankly of little consequence to the logic of the code but they decrease the possibility of making a mistake further along the line. They basically make sure that no misinformative data is allowed to pass futher. In this example we were not able to match the HN and used majority vote deduplication to choose one of the lines so it doesn’t make sense to keep any data in columns like small and big, hence we set them to null. ZIP columns are nulled because relevant information is stored in ZEST_ZIP and so on.